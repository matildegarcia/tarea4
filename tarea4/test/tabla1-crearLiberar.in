# creo una tabla de tamanio 4
crearTTablaJugadores 4

# libero la tabla
liberarTTablaJugadores

# creo una tabla de tamanio 15
crearTTablaJugadores 15

# libero la tabla
liberarTTablaJugadores

Fin