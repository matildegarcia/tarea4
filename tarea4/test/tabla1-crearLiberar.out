1>#  creo una tabla de tamanio 4.
2>3>#  libero la tabla.
4>5>#  creo una tabla de tamanio 15.
6>7>#  libero la tabla.
8>9>Fin.
